packages/bpk-mixins/src/mixins/_forms.scss

```diff
@@ -554,7 +554,7 @@
 
 @mixin bpk-checkbox__input {
   position: absolute;
-  top: ($bpk-spacing-base - $bpk-spacing-md) / 2;
+  top: .175rem;
```

Comment:
Intent gone again?

Reply:
It is for the correct spacing that matches the Sketch libraries and we don't have a token value here for this one